.buildscript/deploy_version.sh

```diff
@@ -17,10 +19,14 @@ elif [ "$TRAVIS_JDK_VERSION" != "$JDK" ]; then
   echo "Skipping snapshot deployment: wrong JDK. Expected '$JDK' but was '$TRAVIS_JDK_VERSION'."
 elif [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
   echo "Skipping snapshot deployment: was pull request."
+elif [ $is_snapshot_version ]; then
+  echo "PR snapshot deployment...'."
+  #./gradlew clean uploadArchives -PnexusUsername="$TRAVIS_USER_NAME" -PnexusPassword="$TRAVIS_PASSWORD"
```

Comment:
sorry, I didn't get this change. We don't deploy anymore or it's ongoing experiment?

Reply:
I'm trying to deploy snapshot versions in PR in maven, it's ongoing still 😓